README.md

```diff
@@ -1 +1,131 @@
-# OOP-school-library
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+
+  <h3><b>School Library</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```

Comment:
- [ ] I noticed that your README doesn't have a LICENSE section but your project does have a LICENSE file. Please consider updating your documentation in order to have an easy way of providing this information to others browsing your project! 🤓 🔧